src/Raven.Server/Config/Categories/StudioConfiguration.cs

```diff
@@ -1,13 +1,78 @@
+using System;
 using System.ComponentModel;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
 using Raven.Server.Config.Attributes;
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.ServerWide;
 
 namespace Raven.Server.Config.Categories
 {
     public class StudioConfiguration : ConfigurationCategory
     {
-        [Description("The directory into which RavenDB will search the studio files, defaults to the base directory")]
+        [Description("The directory in which RavenDB will search the studio files, defaults to the base directory")]
         [DefaultValue(null)]
         [ConfigurationEntry("Studio.Path", ConfigurationEntryScope.ServerWideOnly)]
         public string Path { get; set; }
+        
+        [Description("Semicolon seperated list of notification names which will not be shown in the Studio. If not specified, all notifications are allowed. Example: \"SlowIO;Server_NewVersionAvailable;ClusterTopologyWarning\".")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Studio.Notifications.FilterOut", ConfigurationEntryScope.ServerWideOnly)]
+        public string[] FilteredOutNotifications { get; set; }
+
+        public override void Initialize(IConfigurationRoot settings, IConfigurationRoot serverWideSettings, ResourceType type, string resourceName)
+        {
+            base.Initialize(settings, serverWideSettings, type, resourceName);
+
+            if (type != ResourceType.Server)
+                return;
+
+            ValidateFilteredOutNotificationNames();
+        }
+
+        private void ValidateFilteredOutNotificationNames()
```

Comment:
Let's not do that, we might change this and we don't want to break things if the user has a configuration for new version, frex.